### DISCUSSION TITLE
Personal data marketplace

### DISCUSSION THREAD

**Discussion by F041** [deso-protocol/dips#22] [Medium CIPs]
---
cip: <to be assigned>
title: Personal data marketplace
author:  Gabriele Giordano <@F041>, @gabrielegio on Bitclout
discussions-url: <15>
created: 2021-07-09
---

## One-Line Description
A marketplace for data that otherwise would need more complex ways for obtaining it, like text mining to classify the sex of a user with their posts.

## Explanation and Motivation
With Facebook, users provide a lot of their socio-demo data, like country, languages spoken, age, and not only, that they sell to third parties. 
Developers on Bitclout could integrate those data in a page of profiles, like they do in [CitizenMe](https://www.citizenme.com/), so that interested people (academics, marketers, analysts, etc.) pay for getting those anonymized data. And creators get payed for their *XXI century oil*, **if** they want to provide data. 

The new data will also help Clout_cast to improve the targets for promotions.

## Specification
I think this will imply changes in frontend, core.

## Test Cases
John has an annual salary of 30k$. He puts that data for sale in a section of his profile (next to NFTs?). 
The company X wants to know which brakets of incomes appreciate the most their product. They automatically purchase income data (in creator coin or other means) that cost less than 0.05$ each. 

## Security Considerations
Anonymized data would avoid many problems.

**Comment by HPaulson** [comment]
Anonymized or not -- This is a horrible idea from my perspective. Collecting and selling user data in any way is a major "No", optional or not. The largest benefit of BitClout is that we avoid such action -- even if the user is okay with it to make some extra cash. Additionally, you mention CloutCast (And similar apps) using such for targeted ads -- another concept we 100% should avoid. This seems like very very dangerous territory.

**Reply by AikonMishima** [reply]
In a sense, a part of this data is already viewable to BitClout.com (not the blockchain), as google analytics is installed and thus the anonymized demographics can be viewed by the owner of that google account.

Selling data, however, is a complex matter.

-  In most cases, it's unclear as to how the data gets used and re-identification of anonymous data has proven successful in the past.
- Legal matters like GDPR, etc, also will have an impact on the complexity of what can and can't be disclosed, and how it must be communicated to the Data Subject.

I agree with @HPaulson that this may introduce too many problems for the blockchain.
This can - however - be introduced into future nodes, as a secondary data layer to each profile (with their consent only).

**Reply by HPaulson** [reply]
Yeah I mostly agree, but I don't think sale if data should be encouraged anywhere, even on a third party node

**Reply by bloated-devish** [reply]
If Bitclout becomes big, I think other walled gardens will likely consume BitClout data by just using the identity server and asking users to login with it one time, so they can support their creator. Once they get the PK of their user, they can consume this data as-is from the chain by running their own validator node like anyone else. This is far easier to implement than this CIP.

**Reply by tijno** [reply]
agree with @bloated-devish

When were at 100m DAU there will be dozens of established companies harvesting data from the bitclout blockchain. Not great but it will happen.

There is no need even for bitclout to build this into the platform, even if there were no privacy/policy issues with this.
